packages/gatsby/src/schema/extensions/index.js

```diff
@@ -44,6 +44,11 @@ const typeExtensions = {
       },
     },
   },
+  queryable: {
```

Comment:
Let's call it `@nodeInterface` because this is more semantically correct in this case - we want interface to be "a Node".